### PR TITLE
Remove interpolation on variable attribute

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
```
Warning: Interpolation-only expressions are deprecated

  on main.tf line 15, in resource "exoscale_domain_record" "myserver":
  15:   domain      = "${exoscale_domain.example.id}"
```
Interpolation on variables are now deprecated.
I change all `"${exoscale_domain.example.id}"` -> `exoscale_domain.example.id`

✅  PASS:

- [x]  All acceptance tests

- [x]  All examples

### CODE REVIEW

File: examples/multipart-cloud-init/data.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@ data "template_file" "init" {
   count = length(var.hostnames)
   template = file("cloud-init.yml.tpl")
 
-  vars {
+  vars = {
     fqdn = var.hostnames[count.index]
     ubuntu = "artful"
```
Comment (greut) [inline examples/multipart-cloud-init/data.tf]: bionic?

File: examples/instance-pool/main.tf [hunk 2]
```diff
@@ -7,27 +7,28 @@ variable "template" {
...
+  zone = var.zone
+  name = var.template
 }
 
 resource "exoscale_instance_pool" "instancepool-test" {
   name = "terraforminstancepool"
   description = "test"
-  template_id = "${data.exoscale_compute_template.instancepool.id}"
+  template_id = data.exoscale_compute_template.instancepool.id
   service_offering = "Medium"
```
Comment (falzm) [inline examples/instance-pool/main.tf]: While this is not directly related to your changes, you might as well take this global configuration update opportunity to fix this problem ;)

```console
$ cd examples/instance-pool/
$ terraform init

Initializing the backend...

Initializing provider plugins...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.

$  terraform validate

Error: "service_offering" must be lowercase, got: "Medium"

  on main.tf line 14, in resource "exoscale_instance_pool" "instancepool-test":
  14: resource "exoscale_instance_pool" "instancepool-test" {
```

**Review by falzm — CHANGES_REQUESTED** [review]

File: examples/multipart-cloud-init/data.tf (hunk 1 above)
Comment (pierre-emmanuelJ) [reply examples/multipart-cloud-init/data.tf]: Thanks, I changed for 18.04 in `variables.tf` so you're right

**Review by falzm — APPROVED** [review]
LGTM, thanks @pierre-emmanuelJ 👍
